backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -336,9 +339,21 @@ describe('UserResolver', () => {
           })
 
           // make Peter Lustig Admin
-          const peter = await User.findOneOrFail({ where: { id: user[0].id } })
-          peter.isAdmin = new Date()
-          await peter.save()
+          let peter = await User.findOneOrFail({
+            where: { id: user[0].id },
+            relations: ['userRoles'],
+          })
+          peter.userRoles = [] as UserRole[]
+          peter.userRoles[0] = UserRole.create()
+          peter.userRoles[0].createdAt = new Date()
+          peter.userRoles[0].role = ROLE_NAMES.ROLE_NAME_ADMIN
+          peter.userRoles[0].userId = peter.id
+          await peter.userRoles[0].save()
+
+          peter = await User.findOneOrFail({
+            where: { id: user[0].id },
+            relations: ['userRoles'],
+          })
```

Comment:
```suggestion
```
Why query again?

Reply:
Because of the modifications on:
- new user.userRoles[]
- create new UserRole-entity
- associating the new userRole with the user by setting the userRole.userId with the user.id
- adding the new userRole-entity in the user.userRoles array
- saving the new entity userRole in database, where it becomes it userRole.id value

So on my opinion the user entity with its userRoles-OneToMany relation and vica versa is not initialized correctly still the user is read again.

Reply:
I tried it without the second query and it works.